src/essentials/Colors/ModernColors.ts

```diff
@@ -15,12 +15,17 @@ export const Colors = {
         50: 'hsl(0, 2%, 91%)'
     },
     primary: {
-        1000: 'hsl(346, 51%, 33%)',
+        1100: 'hsl(341, 100%, 13%)',
+        1000: 'hsl(343, 70%, 22%)',
+        950: 'hsl(346, 51%, 33%)',
         900: 'hsl(347, 42%, 43%)',
-        350: 'hsl(349, 89%, 76%)',
-        150: 'hsl(351, 100%, 85%)',
-        100: 'hsl(352, 100%, 93%)',
-        50: 'hsl(9, 100%, 96%)'
+        500: 'hsl(347, 41%, 50%)',
+        350: 'hsl(349, 50%, 65%)',
+        150: 'hsl(348, 50%, 76%)',
+        120: 'hsl(351, 51%, 85%)',
+        100: 'hsl(352, 48%, 91%)',
+        50: 'hsl(350, 46%, 95%)',
```

Comment:
For projects already using these colors this will result in breaking changes, I understand this is aligned with design and it's expected, but I'd still communicate this on the internal slack #dev-frontend to make sure everyone is aware

Reply:
Thanks for the hint @martimalek! Will do!